### PR TITLE
test: confirm sign in tests

### PR DESCRIPTION
**Pull request by Jordan-Nelson**
*Issue #, if available:*

*Description of changes:*
- add confirm sign in tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_authenticator/example/integration_test/pages/authenticator_page.dart [hunk 1]
```diff
@@ -105,6 +107,17 @@ abstract class AuthenticatorPage {
     expect(expectCombinationNotFound, findsOneWidget);
   }
 
+  /// Then I see Invalid verification code
+  Future<void> expectInvalidVerificationCode() async {
+    if (Platform.isAndroid) {
```
Comment (Jordan-Nelson) [inline packages/amplify_authenticator/example/integration_test/pages/authenticator_page.dart:112]: It surprised me that the messages were different on iOS and Android. Seems like this is a bug. Not specific to the authenticator though.

File: packages/amplify_authenticator/example/integration_test/utils/data_utils.dart [hunk 2]
```diff
@@ -98,3 +102,62 @@ Future<AdminCreateUserResponse?> adminCreateUser(
...
+          currentCode
+        }
+      }''';
+
+  Completer<void> subscriptionCompleter = Completer<void>();
+
+  final Stream<GraphQLResponse<String>> operation = Amplify.API.subscribe(
+    GraphQLRequest<String>(document: subscriptionDocument),
+    onEstablished: () {
+      subscriptionCompleter.complete();
```
Comment (haverchuck) [inline packages/amplify_authenticator/example/integration_test/utils/data_utils.dart:130]: I assume you tested and this worked?  When I tried this approach it hung.

**Review by haverchuck — APPROVED** [review]
One question, but otherwise LGTM

File: packages/amplify_authenticator/example/integration_test/pages/authenticator_page.dart (hunk 1 above)
Comment (Jordan-Nelson) [reply packages/amplify_authenticator/example/integration_test/pages/authenticator_page.dart:112]: Here is where the message on Android is coming from - https://github.com/aws-amplify/amplify-android/blob/main/core/src/main/java/com/amplifyframework/auth/AuthException.java#L317

File: packages/amplify_authenticator/example/integration_test/utils/data_utils.dart (hunk 2 above)
Comment (Jordan-Nelson) [reply packages/amplify_authenticator/example/integration_test/utils/data_utils.dart:130]: It hung some (most?) of the time but did seem to work some of the time. I have `await subscriptionCompleter.future` commented out and instead have `await Future.delayed(const Duration(seconds: 1), () {});` as a temp solution.

Comment (haverchuck) [reply packages/amplify_authenticator/example/integration_test/utils/data_utils.dart:130]: Alrighty, thanks

**Pull request merged**